.github/workflows/ci.yml

```diff
@@ -218,7 +218,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        python-version: [3.8, 3.12]
+        python-version: [3.8, 3.12, 3.13]
```

Comment:
Could we also update `release.yml` to release wheel for python3.13 too?

Reply:
done